manifest.json

```diff
@@ -2,7 +2,7 @@
 	"manifest_version": 2,
 	
 	"name": "rikaikun",
-	"version": "0.8.91",
+	"version": "0.8.92",
```

Comment:
Updating this in a PR doesn't really make sense because depending on which features are collected in a release this number will change before publishing.

That being said it's not a big deal because you need some new number for your new options anyway; and it's easy to fix up later before publishing.